src/forwarder/json-rpc-forwarder.js

```diff
@@ -28,6 +28,8 @@ const { getHtsCode, getHtsStorageAt, HTSAddress, LONG_ZERO_PREFIX, getHIP719Code
 /** @type {Partial<import('hardhat/types').HardhatNetworkForkingConfig>} */
 const { url: forkUrl, mirrorNodeUrl, workerPort, localAddresses = [] } = workerData;
 
+require('../slotmap').setLedgerId(workerData.chainId);
```

Comment:
import `setLedgerId` in the `require('..')` that is above.

```suggestion
setLedgerId(workerData.chainId);
```

We should treat exported modules in declated `package.json` as independent. All external used functions/classes/consts should be imported from an `index.js` declared in `package.json`. This way is easier to consumers to also use the package.